SwiftWisdomTests/StandardLibrary/Numbers/CountableClosedRange+ExtensionsTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  CountableClosedRange+ExtensionsTests.swift
+//  SwiftWisdom
+//
+//  Created by Andrew Dolce on 11/21/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+@testable import SwiftWisdom
+
+class CountableClosedRangeExtensionTests: XCTestCase {
+
+    // This test is complex. Here is an explanation as to why:
```

Comment:
totallly agree with the reasoning outlined in these comments, but I think we need to pare it down a little :D